rspace/src/test/scala/coop/rchain/rspace/nextgenrspace/history/HistoryGenerativeSpec.scala

```diff
@@ -60,24 +60,24 @@ class HistoryGenerativeSpec
     finalHistory.runSyncUnsafe(20.seconds).root shouldBe emptyHistory.root
   }
 
-  val arbitraryTestKey32: Arbitrary[TestKey32] =
+  val arbitraryRandomThreeBytes: Arbitrary[Key] =
     Arbitrary(
       Gen
         .listOfN(3, Arbitrary.arbitrary[Int])
-        .map(ints => TestKey32.create(List.fill(29)(0) ++ ints))
+        .map(ints => (List.fill(29)(0) ++ ints).map(_.toByte))
```

Comment:
Only three bytes are arbitrary, as the name says but they are always the last ones. why not having all the 32 bytes random? 

Also, I'm not sure it matters but maybe you should put them in a random position in the 32byte array?

Reply:
the goal is to have interactions between the inserted keys. 32 bytes covers a v large tree. this makes the interactions statistically more probable (hence better coverage)

Reply:
Now I  understand